docs/reference/api/definitions/feature.md

```diff
@@ -9,3 +9,6 @@ Users can extend this class to define their features.
 ::: metaxy.BaseFeature
 
 ::: metaxy.Feature
+
+!!! note "Code Version Access"
```

Comment:
Let's put this in between BaseFeature and Feature. 

Also, let's used > ! notation for consistency with GitHub markdown flavor